mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/Match.kt

```diff
@@ -0,0 +1,69 @@
+package ch.epfl.sdp.mobile.ui.profile
+
+import androidx.compose.foundation.clickable
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.Icon
+import androidx.compose.material.ListItem
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.PawniesIcons
+import ch.epfl.sdp.mobile.ui.SectionSocial
+import ch.epfl.sdp.mobile.ui.i18n.LocalizedStrings
+import ch.epfl.sdp.mobile.ui.social.*
+
+/**
+ * Composes a Match log using a match [title], [subtitle] and an [icon]
+ * @param match a chess Match
+ * @param onMatchClick callback function if clicked on item
```

Comment:
Small inconsistency following renaming

```suggestion
 * @param onClick callback function if clicked on item
```